config/default/jenkins-infra.yaml

```diff
@@ -170,87 +170,46 @@ jenkins:
                   }
                 }
             - script: >
-                multibranchPipelineJob('docker-packer') {
-                  displayName "Packer Docker Build"
-                  branchSources {
+                organizationFolder('Docker Builds') {
+                  description('Docker Builds')
```

Comment:
did you want to add the `PullRequestLabelsBlackListFilterTrait` trait as well?

Reply:
I'll add that in another PR, I'd like to get this in so I can be around to fix if it breaks anything